client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -333,6 +350,9 @@ const MetadataEditor = createClass({
 						<button className='display' onClick={this.toggleThumbnailDisplay}>
 							<i className={`fas fa-caret-${this.state.showThumbnail ? 'right' : 'left'}`} />
 						</button>
+						<button className='display' onClick={this.ThumbnailCapture}>
+							<i className={`fas fa-image`} />
```

Comment:
Do you think that switching to a camera icon is something to consider?  

<img width="56" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/58999374/2f8ca0b3-f6ea-4181-91c5-1feb1f231430">

```
<i class="fas fa-camera"></i>
```
Maybe the `fa-image` icon is more indicative of "open from image library" rather than "generate new picture".  Further, even though none of the other fields/controls currently have one, maybe this should have a tooltip?

Reply:
This is an excellent idea. I will do that.